x-pack/plugins/actions/server/builtin_action_types/jira/index.ts

```diff
@@ -39,6 +39,7 @@ interface GetActionTypeParams {
   configurationUtilities: ActionsConfigurationUtilities;
 }
 
+const DISABLED_ACTION_GROUPS: string[] = ['resolved'];
```

Comment:
I don't think the concept of `resolved` as an action group should exist in the actions plugin, but rather in alerts.
That said, if we _have_ to have it here- we should use a constant (I think there is one in `plugins/alerts/common` already) to ensure the same ID is used across plugins

Reply:
Updated to move everything to the alerts plugin.